WTWSMS/events/religious_events.txt

```diff
@@ -191,6 +199,7 @@ character_event = {
 		OR = {
 			religion = catholic
 			religion = fraticelli
+			religion = nicene
```

Comment:
The reason the Orthodox characters are not in the code are because the game does not simulate individual Pentarchs as their own religious heads outside of their realm. There is not an "ANY_PENTARCH" condition or anything like that.

As each Nicene pentarch cannot dispatch that letter, relying on the Nicene Patriarch is the most accurate equivalent.